easylist_cookie/easylist_cookie_specific_ABP.txt

```diff
@@ -1,2 +1,4 @@
 ! ABP Sepecific fixes
 minhaconexao.com.br#?#section[id][class]:-abp-has(:scope > div[class] > button[class][onclick^="onClickCookiesBannerWeb"])
+! Scripts
+podleze-piekielko.pl#$#abort-on-property-read cookieman
```

Comment:
```suggestion
```

may no work before 2022/2023 year for ABP users with normal subscription.

Reply:
I don't understand? If I load ABP and apply that filter, it works fine.

Reply:
Try load from commit link and test page, not form "my custom added list".